coderd/httpmw/apikey.go

```diff
@@ -216,7 +225,8 @@ func ExtractAPIKey(db database.Store, oauth *OAuth2Configs, redirectToLogin bool
 				})
 				if err != nil {
 					write(http.StatusInternalServerError, codersdk.Response{
-						Message: fmt.Sprintf("API key couldn't update: %s.", err.Error()),
+						Message: signedOutErrorMessage,
```

Comment:
I don't think this error message should say logged out; an internal error happened and we don't know what's wrong, it could be a temporary error like database is down/restarting and the problem could go away after a while (not requiring any action from the user). Another issue is that no matter how many times the user tries to login -- it won't work, which would lead to a bad user experience.

Something like: "Try again or contact the system administrator" could work here `¯\_(ツ)_/¯`.